app/styles/components/c-menu.scss

```diff
@@ -1,142 +1,169 @@
   display: flex;
-  text-transform: capitalize;
+  font-size: 16px;
   font-weight: normal;
+  height: 80px;
   justify-content: space-between;
-  font-size: 16px;
   line-height: 30px;
-  align-items: center;
+  text-transform: capitalize;
+
   ul {
     display: none;
   }
-  .icon_menu_mobile {
+
+  .icon-menu-mobile {
+    cursor: pointer;
     display: block;
     width: 21px;
-    cursor: pointer;
+
   }
-  .img_mobile {
+
+  .img-mobile {
     display: block;
     height: 50px;
   }
-  .img_desktop {
+
+  .img-desktop {
     display: none;
   }
-  .share_header {
+
+  .share-header {
+    color: $color-4;
     display: block;
-    color: $color9;
   }
-  @media screen and (min-width: $screen-m) {
+
+  @media screen and (min-width: $breakpoint-tablet) {
     height: 140px;
-    .img_mobile {
+
+    .img-mobile {
       display: none;
     }
-    .img_desktop {
+
+    .img-desktop {
       display: block;
     }
-    .icon_menu_mobile {
+
+    .icon-menu-mobile {
       display: none;
     }
-    .share_header {
+
+    .share-header {
       display: none;
     }
+
     ul {
-      list-style: none;
       display: block;
+      list-style: none;
+
       li {
         display: inline-block;
         margin-right: 50px;
+
         &:nth-last-child(2) {
           margin-right: 46px;
         }
+
         &:last-child {
-          border-left: 1px solid $color9;
-          padding-left: 46px;
+          border-left: 1px solid $color-4;
           margin-right: 0;
+          padding-left: 46px;
         }
+
         a {
-          color: $color9;
-          text-decoration: none;
-          font-family: $font-family-primary;
-          font-size: $font-size-normal;
-          font-weight: $font-weight-regular;
-          letter-spacing: -0.1px;
+          color: $color-4;
           cursor: pointer;
+          font-family: $font-family-1;
+          font-size: $font-size-normal;
+          font-weight: $font-weight-normal;
+          letter-spacing: -.1px;
+          text-decoration: none;
+        }
+
+        #menu_selected {
+          &::after {
+            background-color: $color-4;
+            border-radius: 100px;
+            content: '';
+            display: block;
+            height: 4px;
+            margin-top: 1px;
         }
-        #menu_selected::after {
-          content: "";
-          display: block;
-          height: 4px;
-          border-radius: 100px;
-          background-color: #ffffff;
-          margin-top: 1px;
         }
       }
     }
+
     .dropdown {
       display: inline-block;
       position: relative;
     }
-    .dropdown_content {
+
+    .dropdown-content {
+      background-color: $color-4;
       display: none;
-      position: absolute;
-      width: 200px;
-      background-color: $color9;
-      z-index: 99;
       padding: 15px;
+      position: absolute;
+      transform: translate(-40%, 0);
       width: 160px;
-      transform: translate(-40%, 0px);
+      z-index: 99;
```

Comment:
This should probably be standardized. See if you can use a round number. Take care of possible issues with the `z-index` of other components such as the modal for example.